install_config/aggregate_logging.adoc

```diff
@@ -476,9 +476,9 @@ request for that size.
 Using NFS storage as a volume or a persistent volume (or via NAS such as
 Gluster) is not supported for Elasticsearch storage, as Lucene relies on file
 system behavior that NFS does not supply. Data corruption and other problems can
-occur. If NFS storage is a requirement, you can allocate a large file on that
-storage to serve as a storage device and treat it as a local mount on each host.
-For example:
+occur. If NFS storage is a requirement, you can allocate a large file on a
+volume to serve as a storage device and mount it locally on one host.
+For example, if your NFS storage volume were mounted at *_/nfs/storage_*:
```

Comment:
alright, no subjunctive then :-)

On Mon, Aug 22, 2016 at 5:10 PM, Ashley Hardin <notifications@github.com>
wrote:

> In install_config/aggregate_logging.adoc
> <https://github.com/openshift/openshift-docs/pull/2703#discussion_r75759467>
> :
>
> > @@ -476,9 +476,9 @@ request for that size.
> >  Using NFS storage as a volume or a persistent volume (or via NAS such as
> >  Gluster) is not supported for Elasticsearch storage, as Lucene relies on file
> >  system behavior that NFS does not supply. Data corruption and other problems can
> > -occur. If NFS storage is a requirement, you can allocate a large file on that
> > -storage to serve as a storage device and treat it as a local mount on each host.
> > -For example:
> > +occur. If NFS storage is a requirement, you can allocate a large file on a
> > +volume to serve as a storage device and mount it locally on one host.
> > +For example, if your NFS storage volume were mounted at *_/nfs/storage_*:
>
> s/were/is
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/2703/files/891d4a26abaff1edd11dd449651785ca97c9c807#r75759467>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABz-l3TRbFe0mGGiTx8PzZZmFUNLGhpks5qihA_gaJpZM4JqSLV>
> .
>